### PR TITLE
Update Github.json

### PR DESCRIPTION
**Pull request by Ajay84sia**
Added GitHub Profile ReadMe Maker

### CODE REVIEW

File: database/Other/Github.json [hunk 1]
```diff
@@ -19,6 +19,13 @@
...
         "category": "Other",
         "subcategory": "github"
+    },
+    {
+        "name": "GitHub Profile ReadMe Maker",
+        "description": "This site will help you to generate markdowns for your GitHub profile",
+        "url": "https://gprm.itsvg.in/",
+        "category": "Other",
+        "subcategory": "github"
     }
```
Comment (rupali-codes) [inline database/Other/Github.json:30]:
```suggestion
```

Comment (Ajay84sia) [reply database/Other/Github.json]: done, extra line? line-30 removed

**Comment by Ajay84sia** [comment]
done, extra line? line-30 removed

**Review by rupali-codes — APPROVED** [review]
thanks

**Pull request merged**